File: Skip_Backend/src/main/java/com/simplife/skip/models/Parada.java
package com.simplife.skip.models;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "parada")
@Entity
public class Parada {

    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "parada_id")
    private Long id;

    @Column(name = "ubicacion")
    private String ubicacion;

    @Column(name = "latitud")
    private float latitud;

    @Column(name = "longitud")
    private float longitud;

    @Column(name = "estado_tabla")
    private boolean estadoTabla;


}

File: Skip_Android/app/src/main/java/com/simplife/skip/models/Parada.kt
package com.simplife.skip.models

import java.io.Serializable

class Parada (
    var ubicacion: String,
    var latitud: Float,
    var longitud: Float

) :Serializable {
    override fun toString(): String {
        return "Parada(ubicacion='$ubicacion', latitud=$latitud, longitud=$longitud)"
    }
}